src/sentry/static/sentry/app/views/alerts/rules/row.tsx

```diff
@@ -49,59 +47,82 @@ class RuleListRow extends React.Component<Props, State> {
-                      "Are you sure you want to delete [name]? You won't be able to view the history of this alert once it's deleted.",
-                      {
-                        name: rule.name,
-                      }
-                    )}
-                    header={t('Delete Alert Rule?')}
-                    priority="danger"
-                    confirmText={t('Delete Rule')}
-                    onConfirm={() => onDelete(slug, rule)}
-                  >
-                    <Button
-                      type="button"
-                      icon={<IconDelete />}
-                      size="small"
-                      title={t('Delete')}
-                    />
-                  </Confirm>
+        <Column>
+          <RuleType>{isIssueAlert(rule) ? t('Issue') : t('Metric')}</RuleType>
+        </Column>
+        <Column>
+          <Title>
+            <Link to={link}>{rule.name}</Link>
+          </Title>
+        </Column>
+        <Column>
+          <ProjectBadge
+            avatarSize={18}
+            project={!projectsLoaded ? {slug} : this.getProject(slug, projects)}
+          />
+        </Column>
+        <Column>
+          <CreatedBy>{rule?.createdBy?.name ?? '-'}</CreatedBy>
+        </Column>
+        <Column>
+          <div>{dateCreated}</div>
+        </Column>
+        <RightColumn>
+          <Access access={['project:write']}>
+            {({hasAccess}) => (
+              <ButtonBar gap={1}>
+                <Confirm
+                  disabled={!hasAccess}
+                  message={tct(
+                    "Are you sure you want to delete [name]? You won't be able to view the history of this alert once it's deleted.",
+                    {
+                      name: rule.name,
+                    }
+                  )}
+                  header={t('Delete Alert Rule?')}
+                  priority="danger"
+                  confirmText={t('Delete Rule')}
+                  onConfirm={() => onDelete(slug, rule)}
+                >
                   <Button
+                    type="button"
+                    icon={<IconDelete />}
                     size="small"
-                    icon={<IconSettings />}
-                    title={t('Edit')}
-                    to={link}
+                    title={t('Delete')}
                   />
-                </ButtonBar>
-              )}
-            </Access>
-          </TableLayout>
-        </AlertRulesPanelItem>
+                </Confirm>
+                <Button
+                  size="small"
+                  icon={<IconSettings />}
+                  title={t('Edit')}
+                  to={link}
+                />
+              </ButtonBar>
+            )}
+          </Access>
+        </RightColumn>
       </ErrorBoundary>
     );
   }
 }
 
+const Column = styled('div')`
+  font-size: ${p => p.theme.fontSizeMedium};
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 100%;
+`;
```

Comment:
We can remove an extra dom element if this is changed to use `css`, see https://github.com/getsentry/sentry/blob/ab0d9dd4aee4a3e73b9c800faf9e5178e29872b9/src/sentry/static/sentry/app/views/settings/incidentRules/list.tsx#L132-L136 for an example.